### ISSUE TITLE
Reshade Depth Buffer not working correctly

### ISSUE THREAD

**Issue by ProphetWild** [yuzu-emu/yuzu#10253]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Affected Build(s)

Mainline 1429

### Description of Issue

Reshade can see the depth buffer in yuzu, specifically Tears of the Kingdom, but it flickers and is not very stable, resulting in things like MXAO or RTGI not working correctly.

### Expected Behavior

Reshade can see a stable depth buffer from yuzu to implement depth based shaders.

### Reproduction Steps

Install latest version of Reshade with support for addons using Vulkan for Yuzu. Activate shader DisplayDepth.fx, click Add-ons menu and select the correct depth buffer. Install and enable quint MXAO or RTGI. The image will flicker.

### Log File

[yuzu_log.txt](https://github.com/yuzu-emu/yuzu/files/11467197/yuzu_log.txt)

### System Configuration

CPU: AMD 5800X3D
GPU/Driver: RTX 4090 531.79
RAM: 32 GB DDR4 3600MHZ
OS: Windows 11

**Comment by junglejuz** [comment]
Yes, I was just about the lodge the same issue. I get the same behaviour, using EA3588. 
To reproduce the isse:
(quint MXSO or RTGI are not needed to reproduce the problem)
Depth buffer is available and visible when selected but constantly flickers to white (losing depth)
![image](https://github.com/yuzu-emu/yuzu/assets/40817462/bf28c7fc-47ee-414e-934e-05fb74e884a0)

**Comment by junglejuz** [comment]
Here's some captures another user recorded of the z-depth and normal passes in action.

https://github.com/yuzu-emu/yuzu/assets/40817462/f3563750-99c6-4cc7-a77d-94499cad9f29
https://github.com/yuzu-emu/yuzu/assets/40817462/fbdd5714-40d0-4843-8d07-0a647becd6e0

**Comment by mathewriver** [comment]
I don't believe this issue will ever be fixed. It's even worse with Zelda Breath of the Wild. Right now I'm trying to get Geforce experience filter RTGI to work with Yuzu, because reshade RTGI is just a no-no for Yuzu.

**Comment by LightMu** [comment]
Try ReShade Addon version, A little better

**Comment by junglejuz** [comment]
Is there any update on whether this can potentially be fixed?

**Comment by markandgo** [comment]
I recently encountered this when trying to use reshade with TOTK. Noticed flickering with the depth buffer.
